### PR TITLE
Initial commit of Windows MSI WiX Toolset

### LINKED ISSUE #58: Determine Windows packaging process for Glimpse
Ideally the build will be automated as described in #2

### PR DESCRIPTION
**Pull request by TrechNex**
Fixes #58 and related to #211 and #180  .

Changes proposed in this pull request:
- WiX Toolset manifest file
- Instructions for how to use it

Still a work in progress, as I need to add back in some old config files/interpreters I removed for the beta testing round

### CODE REVIEW

File: build/windows-glimpse/Glimpse32.wxs [hunk 1]
```diff
@@ -0,0 +1,4533 @@
...
+        <Package Id="*" Keywords="Installer" Description="Glimpse 0.1 Installer" Comments="Glimpse Image Editor is provided under the GNU General Public License version 3." Manufacturer="Glimpse Project" InstallerVersion="100" Languages="1033" Compressed="yes" SummaryCodepage="1252" />
+        <Condition Message="You must be an administrator to install Glimpse Image Editor.">
+            Privileged
+        </Condition>
+        <Condition Message="You must run Windows 7 or later to install Glimpse Image Editor.">
+            VersionNT > 600
+        </Condition>
+        <MajorUpgrade
+          AllowDowngrades = "no"
+          AllowSameVersionUpgrades = "no"
```
Comment (Member1221) [inline build/windows-glimpse/Glimpse32.wxs]: Probably a good idea to set this to "yes".

Comment (TrechNex) [reply build/windows-glimpse/Glimpse32.wxs]: Done 👍

**Comment by Member1221** [comment]
please squash these while merging

**Comment by TrechNex** [comment]
Indeed, it will be squashed. This evening there will probably be more commits as I need to potentially add more directories to the installer. The executable has at least been built though, so it is just polishing off the packaging at this point.

**Comment by TrechNex** [comment]
Now delayed for translation fixes in #222 
I will continue trying to get the WiX stuff here as ready as it can be for release

**Pull request merged**